js/main.js

```diff
@@ -1,10 +1,11 @@
 $("#more").click(function(){
     $("section").css({"position": "fixed"});
-    $("#info").animate({"left": "10%",}, 200);
-    $("#info").animate({"left": "0%",}, 200);
+    $("#info").animate({"left": "0%"}, 200);
```

Comment:
Actually, let's make it synchronous. First the home fades out, then the info slides in. Might make sense to speed things up a bit to make it still seem quick. Thoughts?